packages/wallet/src/redux/sagas/messaging/message-handler.ts

```diff
@@ -108,11 +113,12 @@ function* handleJoinChannelMessage(payload: RequestObject) {
 function* handlePushMessage(payload: RequestObject) {
   // TODO: We need to handle the case where we receive an invalid wallet message
   const {id} = payload;
-  const message = payload.params as JsonRpcMessage;
+  const message = payload.params as PushMessageParams;
   if (isRelayableAction(message.data)) {
     yield put(message.data);
     yield fork(messageSender, outgoingMessageActions.postMessageResponse({id}));
   } else {
+    // TODO: Add schema for PushMessageParams.data to client-api-schema/schema/push-message.json
```

Comment:
I don't think we want to specify the `data` in the JSON schema. The specific messaging format shouldn't really be part of the API that the JSON schema specifies. The client should just blindly pass along `data`.

Reply:
Oh okay, I'll remove the todo.
```suggestion
```